requirements.txt

```diff
@@ -1,19 +1,15 @@
 # Copyright 2020 (c) Cognizant Digital Business, Evolutionary AI. All rights reserved. Issued under the Apache 2.0 License.
 # data science
-pandas==1.1.2
-numpy==1.18.5
-notebook==6.1.5
-scikit-learn==0.23.2
-scipy==1.5.2
-tensorflow==2.2.2
-keras==2.4.3
+pandas==1.5.3
+numpy==1.24.2
+notebook==6.5.3
+scikit-learn==1.2.2
+scipy==1.10.1
+tensorflow==2.11.1
+keras==2.12.0
 neat-python==0.92
-h5py==2.10.0
+h5py==3.8.0
 
 # plotting
-plotly==4.11.0
-matplotlib==3.3.2
-
-# tests
```

Comment:
No longer needed?

Reply:
`nose` is not maintained anymore and doesn't work with Python 3.10. Plain unittest does the job I think:
```
python -m unittest
```
I don't remember why `pytest` was here. I'll add it back later if needed.